flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "skip-arches": ["aarch64"]
```

Comment:
what's the rationale for skipping aarch64?

Reply:
[Something](https://github.com/flathub/flathub/pull/1933#issuecomment-727269231) with freeimage on this arch. I've noticed some other distros skip building on aarch64 as well. Would be nice to investigate and fix though.